ArgoTests/Tests/ExampleTests.swift

```diff
@@ -13,7 +12,7 @@ class ExampleTests: XCTestCase {
 
   func testJSONWithRootObject() {
     let json = JSON.parse <^> JSONFileReader.JSON(fromFile: "root_object")
-    let user: User? = json >>- { $0 <| "user" >>- User.decode }
+    let user: User? = json >>- { ($0 <| "user").value }
```

Comment:
That probably doesn't need to happen for this PR, but it would be nice to look into.